EnglishFilter/sections/foreign.txt

```diff
@@ -4351,6 +4351,10 @@ dap-news.com##.ads
 !------------------------------ Korean -------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/90549
+||haninvegas.com/wp-content/*/img/global/banners/
+haninvegas.com##.banner-header-horz
+haninvegas.com##.banners
```

Comment:
I suppose this `haninvegas.com##div[class^="banner"]` should be ok instead of those two.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/90553/commits/d2cb3d35fa8194acd279a07e8ac367be7b58f98b